signals.go

```diff
@@ -0,0 +1,41 @@
+// +build linux darwin freebsd openbsd solaris netbsd
+
+package main
+
+import (
+	"os"
+	"syscall"
+)
+
+var Signals = []os.Signal{
```

Comment:
I wouldn't propagate the non-standard ones. Also some of these may cause issues if you catch them explicitly. e.g. SIGFPE, or SIGSEGV. The sane ones are [HUP, INT, QUIT, TERM, USR1, USR2]